sdk/sqlmanagement/Microsoft.Azure.Management.SqlManagement/src/Microsoft.Azure.Management.Sql.csproj

```diff
@@ -7,14 +7,9 @@
 		<PackageId>Microsoft.Azure.Management.Sql</PackageId>
 		<Description>Azure SQL Management SDK library</Description>
 		<AssemblyName>Microsoft.Azure.Management.Sql</AssemblyName>
-		<Version>1.37.0-preview</Version>
+		<Version>2.0.0-preview</Version>
 		<PackageTags>Microsoft Azure SQL Management;SQL;SQL Management;</PackageTags>
-		<PackageReleaseNotes>
-			<![CDATA[
-New features:
-- Added support for failover databases readable secondary.
-      ]]>
-		</PackageReleaseNotes>
+		<PackageReleaseNotes>Operations Get, Create, Update and Delete from ManagedInstanceAdministratorsOperations have a different signature. Parameter "administratorName" is omitted.</PackageReleaseNotes>
```

Comment:
Please wrap release notes with `<![CDATA[` and `]]>`, like it used to be. Thanks

Reply:
Done